package.json

```diff
@@ -39,6 +40,7 @@
     "got": "^11.5.2",
     "js-deep-equals": "^2.1.1",
     "mongodb": "^3.6.3",
+    "npm-registry-fetch": "^10.1.1",
```

Comment:
because i've been using pnpm lately, I discovered that we didn't include this dep :D it must have been included by some other dep, so normal npm workflows didn't complain.